src/borg/helpers.py

```diff
@@ -1404,12 +1417,31 @@ def __init__(self):
             if self.logger.level == logging.NOTSET:
                 self.logger.setLevel(logging.WARN)
             self.logger.propagate = False
+        self.emit = self.logger.getEffectiveLevel() == logging.INFO
```

Comment:
perhaps we should document the logging setup and propagation to various parts of the system (progress, remote, implied logging) better => logger.py docstring.